### PR TITLE
Update README.md to have database creation steps and windows variant …

### PR DESCRIPTION
**Pull request by Brodiez**
…commands

Signed-off-by: Brodie <bzoschke@gmail.com>

Contributes to # (add issue number here. i.e. `#302`)

## What did you do?
Updated the README.md to include steps for database creation and windows commands
## Why did you do it?
Ran into set up issues on windows.
## How have you tested it?
Ran on local machine and had the site up with the created account
## Were docs updated if needed?

- [ ] No
- [x] Yes
- [ ] N/A

#### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new console logs
- [ ] Fixes entire issue
- [ ] Partial fix for issue

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,15 +121,35 @@ on how to deploy the project on a live system.
 4. Use the following command if you already have installed the dependencies and want to activate the virtual environment only.
 
 `pipenv shell`
+ 
+5. Use the following commands to create your local database. (If you are using Windows, there will be command variations for the next 3 steps)
 
-5. Run the following command to generate a default user with some pre-existing impacts and locations. This creates an admin user `cfcadmin` with the password `Call4Code`.
+`python3 manage.py makemigrations`
```
Comment (upkarlidder) [inline README.md:127]: Let's change to `./stage1 makemigrations` to be in line with the other non-windows commands.

File: README.md [hunk 2]
```diff
@@ -121,15 +121,35 @@ on how to deploy the project on a live system.
 4. Use the following command if you already have installed the dependencies and want to activate the virtual environment only.
 
 `pipenv shell`
+ 
+5. Use the following commands to create your local database. (If you are using Windows, there will be command variations for the next 3 steps)
 
-5. Run the following command to generate a default user with some pre-existing impacts and locations. This creates an admin user `cfcadmin` with the password `Call4Code`.
+`python3 manage.py makemigrations` 
+
+`python3 manage.py migrate`
```
Comment (upkarlidder) [inline README.md:129]: Let's change to `./stage1 migrate` to be in line with the other non-windows commands.

**Review by upkarlidder — CHANGES_REQUESTED** [review]
Requested some changes. @tonypearson please also review.

File: README.md (hunk 1 above)
Comment (Brodiez) [reply README.md:127]: oh yes, good call

Comment (Brodiez) [reply README.md]: done and pushed

File: README.md (hunk 2 above)
Comment (Brodiez) [reply README.md]: done

**Review by tonypearson — APPROVED** [review]
Looks fine to me.

**Review by upkarlidder — APPROVED** [review]
Thank you @Brodiez for making the changes.

**Pull request merged**